bitnami/metrics-server/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: metrics-server
 sources:
   - https://github.com/bitnami/bitnami-docker-metrics-server
   - https://github.com/kubernetes-incubator/metrics-server
-version: 5.9.3
+version: 5.9.4
```

Comment:
As it is adding a feature, could you bump the minor version?

Reply:
updated